cmd/rqlite/execute.go

```diff
@@ -21,9 +21,9 @@ type executeResponse struct {
 }
 
 func execute(ctx *cli.Context, cmd, line string, argv *argT) error {
-	urlStr := fmt.Sprintf("%s://%s:%d/db/execute?pretty&timings", argv.Protocol, argv.Host, argv.Port)
+	urlStr := fmt.Sprintf("%s://%s:%d%sdb/execute?pretty&timings", argv.Protocol, argv.Host, argv.Port, argv.Prefix)
```

Comment:
Right now, the client expects the API to live at `/`. I'm embedding rqlite (i.e. not using rqlited, but using the go library) into another application at some level that's not the root, e.g. `/rqlite`. This just allows you to specify something besides `/`.